include/faasm/faasm.imports

```diff
@@ -1,33 +0,0 @@
-__faasm_push_state
```

Comment:
This moved to `faasm-toolchain` (along with everything else in the `include/faasm` directory), but it escaped deletion from this repo.

It only became a problem when adding a new function to `host_interface.h`, and the linting was picking up the version in this repo, rather than the one installed by `faasm-toolchain`